### ISSUE TITLE
[css-contain] `container-type` shouldn't allow mixing `normal` with other values

### ISSUE THREAD

**Issue by Loirooriol** [w3c/csswg-drafts#7669]
https://drafts.csswg.org/css-contain-3/#container-type

> [Value:](https://www.w3.org/TR/css-values/#value-defs) `normal` [||](https://drafts.csswg.org/css-values-4/#comb-any) [ `size` [|](https://drafts.csswg.org/css-values-4/#comb-one) `inline-size` ]

But `normal size` and `normal inline-size` don't make sense.

`normal` establishes a query container for [container style queries](https://drafts.csswg.org/css-contain-3/#container-style-query), but that's the default, included in `size` and `inline-size`:

> Unless otherwise noted, all elements are [query containers](https://drafts.csswg.org/css-contain-3/#query-container) for the purpose [container queries](https://drafts.csswg.org/css-contain-3/#container-query) that do no require explicit containment (such as [container style queries](https://drafts.csswg.org/css-contain-3/#container-style-query))

The definitions are even contradictory, e.g. with `normal size`, both of these apply:

> The element is not a [query container](https://drafts.csswg.org/css-contain-3/#query-container) for any [container size queries](https://drafts.csswg.org/css-contain-3/#container-size-query)

> Establishes a [query container](https://drafts.csswg.org/css-contain-3/#query-container) for [container size queries](https://drafts.csswg.org/css-contain-3/#container-size-query)

So the syntax should just be `normal | size | inline-size`.

**Comment by Loirooriol** [comment]
@mirisuzanne Seems a mistake from 3aed70b094c3ca92eb7d2df1dc49198b8e546031, it was

```
none | [ style || [ size | inline-size ] ]
```

and then `style` was removed (included in the others), so it should be

```
none | [ size | inline-size ]
```

**Comment by mirisuzanne** [comment]
@Loirooriol should be fixed in [964c07f](https://github.com/w3c/csswg-drafts/commit/964c07fd1f31f26b0d1629858cb4a0ac6dc8bf14).

**Comment by mirisuzanne** [comment]
(It looks to me like the WP Tests are accurate already)
